src/synthesizer/instruments/__init__.py

```
@@ -1,11 +1,20 @@
 from synthesizer.instruments.filters import UVJ, Filter, FilterCollection
 from synthesizer.instruments.instrument import Instrument
 from synthesizer.instruments.instrument_collection import InstrumentCollection
+from synthesizer.instruments.premade import *
```

Ruff (0.8.2)

`from synthesizer.instruments.premade import *` used; unable to detect undefined names

(F403)